templates/common/_base/files/proxy.yaml

```diff
@@ -0,0 +1,15 @@
+mode: 0600
+path: "/somewhere/proxy"
```

Comment:
this isn't obviously working (yet) @cgwalters any good location to drop this file at? :thinking:

Reply:
If you were to put it at `/etc/environment` you might not even have to explicitly specify `EnvironmentFile` on the units.

Maybe it's better to be explicit here though, and reference it clearly visibly, though.